### PR TITLE
GUI-1966 Update layout of form fields and labels.

### PR DESCRIPTION
**Pull request by coyote240**
Set left and right large column widths on form fields and labels.
https://eucalyptus.atlassian.net/browse/GUI-1966

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Can one of the admins verify this patch?

File: eucaconsole/templates/ipaddresses/ipaddresses.pt [hunk 1]
```diff
@@ -121,7 +121,9 @@
                         <div tal:condition="is_vpc_supported" class="small-6 columns value" i18n:translate="">VPC</div>
                     </div>
                 </div>
-                ${panel('form_field', field=allocate_form.ipcount, leftcol_width=6, rightcol_width=6, **html_attrs)}
+                ${panel('form_field', field=allocate_form.ipcount,
```
Comment (kamalgill) [inline eucaconsole/templates/ipaddresses/ipaddresses.pt]: @coyote240: There's a bug in the version of Chameleon in our target deployment environment that requires the ${panel...} calls to be on a single line, so we'll need to remove the line breaks here.  Once we move to CentOS 7 and Python 2.7, we can add the line breaks to the panel calls.

**Comment by aliceh** [comment]
ok to test

**Comment by aliceh** [comment]
test this please

**Comment by coyote240** [comment]
Hal, test this please

**Comment by aliceh** [comment]
Guitester, retest this please

**Comment by coyote240** [comment]
Please test using both AWS and Euca as there are field type differences between the two.

**Pull request merged**